pkg/webhook/webhook.go

```diff
@@ -88,6 +70,15 @@ func (pwh *pravegaWebhookHandler) mutatePravegaManifest(ctx context.Context, p *
 }
 
 func (pwh *pravegaWebhookHandler) mutatePravegaVersion(ctx context.Context, p *pravegav1alpha1.PravegaCluster) error {
+	configMap := &corev1.ConfigMap{}
+	err := pwh.client.Get(ctx, types.NamespacedName{Name: p.Spec.SupportedUpgradePaths, Namespace: p.Namespace}, configMap)
+	if err != nil {
```

Comment:
It is important to distinguish between error in client.Get and exists check for config map.
Here is how we can check if the config map exists:
https://github.com/pravega/pravega-operator/blob/master/pkg/controller/pravegacluster/pravegacluster_controller.go#L93

If config map is not found, please provide a suitable message to the user to create this config map. Current message does not explain if the user can do anything to fix this problem.

Reply:
updated